pages/common/dolt-fetch.md

```diff
@@ -0,0 +1,16 @@
+# dolt fetch
+
+> Download objects and refs from another repository.
+> More information: <https://github.com/dolthub/dolt>.
+
+- Fetch refs and update remote-tracking branches from the default remote named "origin":
+
+`dolt fetch`
+
+- Fetch latest changes from a specific remote upstream repository:
+
+`dolt fetch {{remote_name}}`
+
+- Update refs to remote branches with the current state of the remote, overwriting any conflicting history:
```

Comment:
Perhaps this better
```suggestion
- Update branches with the current state of the remote, overwriting any conflicting history:
```

Reply:
solved in 95bb1b6bf7274caad847319db358b771a1736bcf